app/views/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+<% @users.each do |user| %>
+<a href="/users/<%=user.id %>">
+<div class="image"><img src="<%= @user.photo%>" alt="<%= @user.name%>"></div>
```

Comment:
- [ ] @FaranoshAmini great work implementing the changes requested, however, you used @user.photo and @user.name inside a loop resulting in the error below, kindly remove the ` @ ` to remove the error
![image](https://user-images.githubusercontent.com/61408860/230737116-6891e017-6e5b-482b-9929-629e4ceb3fb5.png)